frontend/app/chat/[chatId]/components/MessagesDialog/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { useTranslation } from "react-i18next";
+
+import { useChatContext } from "@/lib/context";
+
+import { ChatMessage } from "./components";
+import { ChatNotification } from "./components/ChatNotification/ChatNotification";
+import { useMessagesDialog } from "./hooks/useMessagesDialog";
+import { getMergedChatHistoryWithReducedNotifications } from "./utils/getMergedChatHistoryWithReducedNotifications";
+
+export const MessagesDialog = (): JSX.Element => {
```

Comment:
ChatDialogue instead of MessagesDialog 
N.B: Referring to Dialogue (history of messages + notifications) and not to Dialog as a dialog box